src/types/fieldExtractor.ts

```diff
@@ -1,13 +1,30 @@
-import { getTrait, NFT } from './nft';
+import { getTrait, NFT, NftFactory } from './nft';
 
-export enum CustomFieldExtractor {
+export type ExtractorTypes = {
+  title?: TitleExtractorTypes
+}
+
+export enum TitleExtractorTypes {
   METADATA_NAME = 'metadata.name',
   ATTRIBUTES_TRAIT_SONG_TITLE = 'attributes.trait.songTitle',
-  ATTRIBUTES_TRAIT_TRACK = 'attributes.trait.track'
+  ATTRIBUTES_TRAIT_TRACK = 'attributes.trait.track',
+  DEFAULT = ATTRIBUTES_TRAIT_TRACK
```

Comment:
i'm not sure this is a great default - it's only really legacy and used in src/db/migrations/11-add-daniel-allan-glasshouse.ts

actually i think the only time multi-track-multiprint-contract is used other than in this PR is for glasshouse.

having said that, i do think we want to evolve multi-track-multiprint-contract to become the primary/most commonly used mapper since it should cover a superset of most nft contracts.

given that, i'd say lets:
 - avoid doing a default for now, throw an error if there is no extractor specificed
 - update daniel allen retroactively to make it use the ATTRIBUTES_TRAIT_TRACK title extractor